data.sql

```diff
@@ -8,3 +8,15 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Pikachu', '01/07/2021', 15.04,false, 1 );
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Devimon', '05/12/2017', 11, true, 5);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Charmander', '08/02/2020', 11, false, 0);
```

Comment:
- [ ] The weight for this animal should be a negative value `-11.0` I am suggesting you update it.